pinot-tools/src/main/java/com/linkedin/pinot/tools/segment/converter/SegmentMergeCommand.java

```diff
@@ -112,6 +112,23 @@ public boolean execute() throws Exception {
       workingDir = new File(_workingDirectory);
     }
 
+    // Check if the input path is directory or comma separated files
+    String[] paths = _inputSegmentPaths.split(INPUT_SEGMENT_SEPARATOR);
```

Comment:
Should we support nested directory or passing multiple directories?

Reply:
Since I support untarred segment file, I think that we should not support nested directory. I will update the code to support multiple paths and/or multiple directories (1-level).